applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_FIC_element.cpp

```diff
@@ -290,10 +299,8 @@ void UPwSmallStrainFICElement<2, 4>::ExtrapolateGPConstitutiveTensor(const array
     const SizeType Dim      = 2;
     const SizeType NumNodes = 4;
 
-    BoundedMatrix<double, NumNodes, NumNodes> ExtrapolationMatrix;
-    this->CalculateExtrapolationMatrix(ExtrapolationMatrix);
-
-    auto const VoigtSize = this->GetStressStatePolicy().GetVoigtSize();
+    const auto ExtrapolationMatrix = CalculateSquareExtrapolationMatrix(this);
+    auto const VoigtSize           = this->GetStressStatePolicy().GetVoigtSize();
```

Comment:
Although syntactically correct, in our code base, we put `const` in front of the (deduced) type name:
```suggestion
    const auto VoigtSize           = this->GetStressStatePolicy().GetVoigtSize();
```
There are more such occurrences in this file. Can you please change them, too? Thanks.

Reply:
corrected